airflow/providers/amazon/aws/operators/redshift_to_s3.py

```diff
@@ -107,7 +112,12 @@ def execute(self, context):
         credentials = s3_hook.get_credentials()
         unload_options = '\n\t\t\t'.join(self.unload_options)
         s3_key = '{}/{}_'.format(self.s3_key, self.table) if self.table_as_file_name else self.s3_key
-        select_query = "SELECT * FROM {schema}.{table}".format(schema=self.schema, table=self.table)
+        if self.custom_select_query is None:
+            if self.schema is None or self.table is None:
+                raise AirflowBadRequest("schema, table set to None")
```

Comment:
I think is better to use `Both custom_select_query and (schema, table) be None not allow here` or some else, you could not just hint with `schema` and `table`

Reply:
@swiftomkar WDYT, about this one?